cmd/root.go

```diff
@@ -27,6 +28,15 @@ var rootCmd = &cobra.Command{
 		"Users can easily launch an instance with or without custom configurations.",
 }
 
+var versionCmd = &cobra.Command{
```

Comment:
Let's make `version` a full-fledged subcommand. That would involve moving this cobra command to a separate file, maybe `version.go`, rather than coupling it with the `launch` command.

Reply:
Good point. Made the changes needed for this